src/Raven.Server/RavenServer.cs

```diff
@@ -379,20 +397,43 @@ public DynamicJsonValue ToJson()
                     [nameof(CurrentConsumption)] = CurrentConsumption,
                     [nameof(MachineCpuUsage)] = MachineCpuUsage,
                     [nameof(History)] = historyByMinute,
+                    [nameof(LastSyncTime)] = LastSyncTime
                 };
             }
         }
 
         private void StartMonitoringCpuCredits()
         {
             CpuCreditsBalance.Used = true;
-            CpuCreditsBalance.RemainingCpuCredits = CpuCreditsBalance.BaseCredits;
             CpuCreditsBalance.BackgroundTasksThresholdReleaseValue = CpuCreditsBalance.BackgroundTasksThreshold * 1.25;
             CpuCreditsBalance.FailoverThresholdReleaseValue = CpuCreditsBalance.FailoverThreshold * 1.25;
             CpuCreditsBalance.CreditsGainedPerSecond = CpuCreditsBalance.BaseCredits / 3600;
 
             int remainingTimeToBackgroundAlert = 15, remainingTimeToFailvoerAlert = 5;
             AlertRaised backgroundTasksAlert = null, failoverAlert = null;
+
+            if (string.IsNullOrEmpty(Configuration.Server.CpuCreditsExec) == false)
```

Comment:
Here you are checking string is null or empty, few lines up only for null. Can we check null or empty in both places?